.github/workflows/coverage.yml

```diff
@@ -9,10 +9,10 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-node@v1
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
```

Comment:
can updates to existing workfolows go into a separate PR, please?

Reply:
I reverted the action version updates in https://github.com/ExodusMovement/schemasafe/pull/166/commits/aba7d81ad68c0d3c5970120b8acd7ad79fce4919 (kept the node version bump though because it's either that or using the `--ignore-engines` flag when installing)